### ISSUE TITLE
Update what's new on storefront

### ISSUE THREAD

**Issue by BeckyBrekke** [equinor/design-system#1988]
Update what's new on front page. 
Can copy the text from Slack release message this week.

**Comment by BeckyBrekke** [comment]
@lucasveil Can you check the What's new that I added? Is it too wordy? Feel free to edit it. 
https://zeroheight.com/0b0c666ab/v/latest/p/238bf3-equinor-design-system
Slack post: https://equinor.slack.com/archives/CJT20H1B9/p1645196037105879

**Comment by lucasveil** [comment]
Looks good @BeckyBrekke 
You kept the link "All our latest news" and it is perfect for the purpose :)
